### PR TITLE
travis and circle ci

### PR DESCRIPTION
**Pull request by cdflynn**

### CODE REVIEW

File: zetautils/src/main/java/zeta/android/utils/lang/StringUtils.java [hunk 1]
```diff
@@ -131,7 +133,7 @@ public static String capitalizeFirstLetter(@Nullable final String str) {
             return EMPTY_STRING;
         }
 
-        String tmpStr = str.toLowerCase();
+        String tmpStr = str.toLowerCase(Locale.getDefault());
```
Comment (manjunathc23) [inline zetautils/src/main/java/zeta/android/utils/lang/StringUtils.java:136]: 👍

**Comment by manjunathc23** [comment]
Changes looks good to me, We can merge this once the build is completed. Thanks @cdflynn

**Comment by manjunathc23** [comment]
This was merged into master! Thanks @cdflynn

**Comment by cdflynn** [comment]
Thanks @manjunathc23 !

**Pull request merged**